portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,196 @@
+                        <img class="card-img-top" src="/images/eben.png" alt="Card image cap">
+                        <div class="box">
+                            <i class="fas fa-code"></i>
+                            <div class="card-body">
+                            <div class="text">Currency converter</div>
+                            <p class="par">This is an app that converts US dollars to Ghanaian cedis.
+                            </p>
+                            <a href="https://play.google.com/store/apps/details?id=currency.com.slide" target="_blank" class="btn btn-primary">Go check it out</a>
+                            </div>
+                        </div>
+
+                    </div>
+
+
+
+
+
+
+                </div>
+
+            </div>
+
+        </section>
+
+
+
+
+        <!--skills section start -->
+        <section class="skills" id="skills">
+            <div class="max-width"> 
+                <h2 class="title">My Skills</h2>
+                <div class="skills-content">
+                    <div class="column left">
+                        <div class="text">My Professional Skills</div>
+                        <p>I am familiar with the programming languages on the right. My level of proficiency is based
+                            on my familiarity with each language and that includes frameworks associated with the language
+                            and the number of projects I have built with the language.
+                        </p>
+                    </div>
+                    <div class="column right">
+                        <div class="bars">
+                            <div class="info">
+                                <span>HTML</span>
+                                <span>80%</span>
+                                
+                            </div>
+                            <div class="line html"></div>
+                        </div>
+                        
+
+                        
+                        <div class="bars">
+                            <div class="info">
+                                <span>Java</span>
+                                <span>50%</span>
+                                
+                            </div>
+                            <div class="line java"></div>
+                            
+                            
+                        </div>
+                        
+                        <div class="bars">
+                            <div class="info">
+                                <span>Python</span>
+                                <span>70%</span>
+                                
+                            </div>
+                            <div class="line python"></div>
+                            
+                        </div>
+                    </div>
+
+                    </div>
+
+            </div>
+
+        </section>
+
+
+
+
+
+
+
+
+
+        
```

Comment:
I think you normally want to include script as early as possible in your html file